detekt-rules/src/test/kotlin/OrtPackageNamingTest.kt

```diff
@@ -154,7 +154,7 @@ private fun TestConfiguration.createFile(dir: String, content: String): File {
     val parent = File("build/classes/kotlin/test/$dir").safeMkdirs()
 
     afterTest {
-        parent.safeDeleteRecursively(force = true)
```

Comment:
commit-message: Something is missing in the title? "Remove the force argument **a** delete function "